src/pybind/mgr/cephadm/module.py

```diff
@@ -92,9 +93,9 @@ def os_exit_noop(status: int) -> None:
 
 
 # Default container images -----------------------------------------------------
-DEFAULT_IMAGE = 'quay.io/ceph/ceph'
-DEFAULT_PROMETHEUS_IMAGE = 'quay.io/prometheus/prometheus:v2.33.4'
-DEFAULT_NODE_EXPORTER_IMAGE = 'quay.io/prometheus/node-exporter:v1.3.1'
+DEFAULT_IMAGE = 'docker.io/rhcsdashboard/ceph-exporter'
```

Comment:
Why not a container image for this daemon type instead of changing the default ceph image we're using?

Reply:
We're not merging this into master directly, as the image will keep on changing in this branch as long as we introduce new changes on C++ side and once feature branch is merged in master it'll be the part of default ceph image.